src/components/login/LoginForm.js

```diff
@@ -80,3 +80,15 @@ const LoginForm = ({
 );
 
 export default withStyles(style)(LoginForm);
+
+/** Various props that control how the user interacts with
+ *  the login form
+ * @typedef ILoginFormComponentProps
```

Comment:
You should also change the above type notation for LoginForm to 
```
* @param {ILoginFormComponentProps} props
```

Reply:
Fixed

Reply:
I think you forgot to push